packages/e2e/features/ui/components/authenticator/sign-up-with-phone.feature

```diff
@@ -7,13 +7,14 @@ Feature: Sign Up with Phone
     And I click "Create account"
 
 
-@next @react @vue @angular
+  @angular @next @react @vue
   Scenario: Login mechanism set to "phone_number"
     Then I see "Phone Number" as an input field
     And I don't see "Username" as an input field
     And I don't see "Email" as an input field
 
-@next @react @vue @skip
+  # Sign up tests skipped due to SES limits
```

Comment:
Do we have an answer for this?  I assume it's the same thing as customers:
https://docs.aws.amazon.com/ses/latest/dg/manage-sending-quotas-request-increase.html

Which means, should this be documented for them too?